src/Camera.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.camera.Camera" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../../../xsd/widget.xsd">
+    <name>Camera</name>
+    <description>Allows users to take and save pictures on their mendix applications</description>
+    <icon>
+        iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAQAAADZc7J/AAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAAmJLR0QAAKqNIzIAAAAJcEhZcwAADdcAAA3XAUIom3gAAAAHdElNRQfhCgwJFhuIhoIqAAABhUlEQVRIx+3UP2sUURQF8N9kXVgLG5EQUxjShZDCKSRgwE+wyhb5BEkXkm3sxDKN1YKdlWAjdkFMb2WCSwgZ0TJlAlqJsNGw2bHRdf68iQPbifc2dw7nnvPmvjvD/4gy9Q1PNOx5hlXtEvfMlmERvJKpr7qt4TNanroZsHvveb1zbUmDeZwzHL9CxwOzOfSO6xXS+76O6xOv7bBZ4VYvNzmcSOAwMsrdRDbOvdKXmBG7bxF88Ng9D8ecVKX2gSXM6VjW0rRtKLWGyJcMr7K9acH+r6eBrsi61I5r2jlmsP2HJQsGOayHXanzAjco8IKx++8cWRGXuVPB8fXNWS5tzKrEoEgNCyTiABq7kNQTmHEaQE8pbGylQOzIWQndM+1WmRwa4kdN3QL2SctGgFuxB9siPaNM+6J53+oLDK1jRc9bL3W1zDsIMi/5UHbFGmDaRtA9lUZSl8VAYjY0uj/rcVFxE/ViNKU/QTv9hu86lX+Ev/p7FOGudnnDasSJN95NdP5/JH4CyctNHfjcPXwAAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMTAtMTJUMDk6MjI6MjcrMDI6MDCMRiaFAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTEwLTEyVDA5OjIyOjI3KzAyOjAw/RueOQAAABl0RVh0U29mdHdhcmUAd3d3Lmlua3NjYXBlLm9yZ5vuPBoAAAAASUVORK5CYII=
+    </icon>
+    <properties>
+    <property key="widthUnit" type="enumeration" defaultValue="pixels">
+            <caption>Width unit</caption>
+            <category>Appearance</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="percentage">Percentage</enumerationValue>
+                <enumerationValue key="pixels">Pixels</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="width" type="integer" defaultValue="714">
+            <caption>Width</caption>
+            <category>Appearance</category>
+            <description/>
+        </property>
+        <property key="heightUnit" type="enumeration" defaultValue="pixels">
+            <caption>Height unit</caption>
+            <category>Appearance</category>
+            <description>'Percentage of width' is the aspect ratio, 'Pixels' is absolute.</description>
```

Comment:
You removed the warning. Unless it's not true anymore, please re-add it.
`'Percentage of width' is the aspect ratio, 'Pixels' is absolute. Warning: When using 'Percentage of parent' the parent container must have an absolute height, else nothing is displayed.`

Reply:
Am not using the percentage of parent anymore since it is complicated to work with especially with webcamjs library that requires the parent element to have a defined dimensions before it can render the webcamera.